full/Dockerfile

```
@@ -204,41 +204,57 @@ ENV PATH=$PATH:/home/gitpod/.nvm/versions/node/v${NODE_VERSION}/bin
 LABEL dazzle/layer=lang-python
 LABEL dazzle/test=tests/lang-python.yaml
 USER gitpod
-ENV PATH=$HOME/.pyenv/bin:$HOME/.pyenv/shims:$PATH
-RUN curl -fsSL https://github.com/pyenv/pyenv-installer/raw/master/bin/pyenv-installer | bash \
+
+ENV GP_PYTHON_VERSION=3.7.7 \
+    GP_PYTHON_SHORT_VERSION=3.7 \
+    PATH=$HOME/.pyenv/bin:$HOME/.pyenv/shims:/home/gitpod/.local/bin:$PATH
+
+RUN sudo apt-get update \
+    && sudo apt-get install -yq libssl-dev zlib1g-dev libbz2-dev libreadline-dev libsqlite3-dev llvm libncurses5-dev libncursesw5-dev xz-utils \
+    && curl -fsSL https://www.python.org/ftp/python/${GP_PYTHON_VERSION}/Python-${GP_PYTHON_VERSION}.tar.xz -o Python-${GP_PYTHON_VERSION}.tar.xz \
+    && tar xsf Python-${GP_PYTHON_VERSION}.tar.xz \
+    && cd Python-${GP_PYTHON_VERSION} \
+    && ./configure \
+        #--enable-optimizations \
```

Note: this needs to be made before this is merged, **and is very much worth it but slows down the build a ton**.

```suggestion
        --enable-optimizations \
```